MapleServer2/Tools/LuaLoader.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using MoonSharp.Interpreter;
+using NLog;
+
+namespace MapleServer2.Tools
+{
+    public class LuaLoader
```

Comment:
Can you abstract this to just `ScriptLoader` rather than hardcoding the script language?